content/_en/policy/rules-of-use.md

```diff
@@ -32,15 +32,20 @@ The Login.gov service employs recognized security and privacy best practices.
 
 Prior to using the Login.gov service, you are required to agree to these terms of service. When the Login.gov service changes its terms of service, you will be given the option to agree or to decline the updated terms of service. Similarly, when conducting certain activities in the Login.gov service, such as providing personal information for the first time, the Login.gov service may require you to re-confirm your understanding of how it uses your information.
 
-# 2. Your Agreement
+## 2. Your Agreement
 
 In addition to details about your consent stated elsewhere in these terms, by accepting these terms and using the Login.gov service, you agree that:
 
 1. You are not a child under 13 years of age,
+
```

Comment:
The extra newlines add wrapping paragraphs to the list item contents. Maybe we should style the lists instead, also so that they're consistent site-wide? 

In fact, it looks like we might just need to add `ol` to the selector here, since we have some of the desired line-height styling for lists already:

https://github.com/18F/identity-site/blob/049f48de2c6043f8aad2ddea0cfbb359420e194d/_sass/components/_typography.scss#L30

Reply:
@aduth Added `ol` to the selector.